server/container/guice/protocols/webadmin-mailbox/src/main/java/org/apache/james/modules/server/MailboxRoutesModule.java

```diff
@@ -58,5 +70,21 @@ protected void configure() {
         Multibinder.newSetBinder(binder(), TaskRegistration.class, Names.named(ALL_MAILBOXES_TASKS));
         Multibinder.newSetBinder(binder(), TaskRegistration.class, Names.named(ONE_MAILBOX_TASKS));
         Multibinder.newSetBinder(binder(), TaskRegistration.class, Names.named(ONE_MAIL_TASKS));
+
+        bind(Zipper.class).in(Scopes.SINGLETON);
+        bind(ArchiveService.class).to(Zipper.class);
+
+        bind(ZipMailArchiveRestorer.class).in(Scopes.SINGLETON);
+        bind(MailArchiveRestorer.class).to(ZipMailArchiveRestorer.class);
+
+        bind(ZipArchivesLoader.class).in(Scopes.SINGLETON);
+        bind(MailArchivesLoader.class).to(ZipArchivesLoader.class);
+
+        bind(DefaultMailboxBackup.class).in(Scopes.SINGLETON);
+        bind(MailboxBackup.class).to(DefaultMailboxBackup.class);
```

Comment:
Can we move all zipper related stuff to another Abstract module that we `install()` here?

It would make a future split easier.